thesis/03-implementation.tex

```diff
@@ -175,24 +174,24 @@ \subsection{Components overview}
 
 \subsubsection{Frontend}
 
-The web application contains two main subcomponents: \textit{Guide Creator} and \textit{Guide Viewer}, responsible respectively for: creating instructions and viewing animated guides. Other pages are mainly related to the community aspect and will only be covered superficially in this section.
+The web application contains two main subcomponents: \textit{Guide Creator} and \textit{Guide Viewer}, responsible respectively for creating instructions, and viewing animated guides. Other components are mainly related to the community aspect and will only be covered superficially.
 
 \begin{figure}[H]
   \caption{Overview of the frontend pages and components. Grayed out pages are related to user management.}
   \centering
     \includegraphics[width=\textwidth]{assets/3-frontend-architecture.png}
 \end{figure}
 
-The web application has been structured in a way that encourages building and reusing small components. An example would be a \textit{Viewer} component which is responsible for rendering origami figures and appears in both \textit{Guide Creator} and \textit{Guide Viewer} pages. The structure has allowed us to keep the code maintainable and implement changes quickly.
+The web application has been structured in a way that encourages building and reusing small components. An example would be a \textit{Viewer} component which is responsible for rendering origami figures and appears in both \textit{Guide Creator} and \textit{Guide Viewer} pages. This structure has allowed us to keep the code maintainable and implement changes quickly.
```

Comment:
a \textit{Viewer} component -> the \textt{Viewer} component?

Personal nitpick: `the structure` IMO sounds better than `this structure`.